src/create-jobs.js

```diff
@@ -157,6 +157,48 @@ function checks (config) {
                 return createQaJobs('./vendor/bin/phpbench run --revs=2 --iterations=2 --report=aggregate', config);
             }
         ),
+        new Check(
+            config.code_checks,
+            [
+                function () {
+                    const composerFile = JSON.parse(fs.readFileSync('composer.json'));
```

Comment:
I would prefer having this added to the first line of the `check` function so it can be reused in both `infection` and `roave-infection-static-analysis-plugin` check